src/legacy/store/GraphQLStoreChangeEmitter.js

```diff
@@ -22,11 +21,10 @@ export type ChangeSubscription = {
   remove: () => void;
 };
 
-var batchUpdate = callback => callback();
-var subscribers = [];
-
-var executingIDs = {};
-var scheduledIDs = null;
+type Subscriber = {
+  callback: Function,
```

Comment:
Good catch :+1: `() => void` could probably become its own `SubscriptionCallback` type at this point now too...